src/pages/dashboard/index.page.tsx

```diff
@@ -800,32 +800,32 @@ const Dashboard = (): any => {
         </Stack>
         {i18n.language === 'nl' && <NewsletterSignupForm />}
         <Footer />
+        <Modal
+          variant={ModalVariants.QUESTION}
+          visible={isModalVisible}
+          onConfirm={async () => {
+            deleteItemByIdMutation.mutate({
+              id: parseOfferId(toBeDeletedItem['@id']),
+            });
+          }}
```

Comment:
```suggestion
          onConfirm={() => {
            deleteItemByIdMutation.mutate({
              id: parseOfferId(toBeDeletedItem['@id']),
            });
          }}
```

Reply:
@simon-debruijn If I remove it I get the same issue with query invalidation as described in the ticket, it's like it awaits for the mutation but not onSuccess. I've tried a few combination with mutate/mutateAsync etc but I can't really grasp what makes it work or not or what is wrong with our setup because yes it should work as is, but could be the whole query hydration logic? Not sure :s